muxer.go

```diff
@@ -285,16 +290,12 @@ func (m *Muxer) retransmitTables(force bool) (int, error) {
 func (m *Muxer) WriteTables() (int, error) {
 	bytesWritten := 0
 
-	if m.patBytes.Len() != m.packetSize {
-		if err := m.generatePAT(); err != nil {
-			return bytesWritten, err
-		}
+	if err := m.generatePAT(); err != nil {
```

Comment:
What's the reasoning behind removing this condition ?

Reply:
Hello, to increase the continuity counter, PAT and PMT needs to be regenerated during every `WriteTables()`, therefore they can't be cached anymore. That condition allowed to cache PAT and PMT, this is not possible anymore.